Source/WebKit/UIProcess/UserMediaProcessManager.cpp

```diff
@@ -40,6 +40,8 @@ static const ASCIILiteral videoExtensionPath { "com.apple.webkit.camera"_s };
 static const ASCIILiteral appleCameraServicePath { "com.apple.applecamerad"_s };
 static const ASCIILiteral additionalAppleCameraServicePath { "com.apple.appleh13camerad"_s };
 static const ASCIILiteral appleCameraUserClientPath { "com.apple.aneuserd"_s };
+static const ASCIILiteral appleCameraUserClientIOKitClientClass { "H11ANEInDirectPathClient"_s };
+static const ASCIILiteral appleCameraUserClientIOKitServiceClass { "H11ANEIn"_s };
```

Comment:
Nit: I still don't understand why we have these string constants declared in more than one place.

Reply:
This was used for capturing in WebProcess.
These changes are probably not needed now that we are only capturing in GPUProcess.
I'll remove this code path entirely in a separate PR.